ios/Classes/BetterPlayer.m

```diff
@@ -1066,26 +1097,26 @@ - (void) hideLimitedPlanCoverViewInPIP {
     }
 }
 
-/// Hide LimitedPlanCoverView after the PIP is completely gone.
+/// Check when the PIP is completely gone.
```

Comment:
This function is mainly "Do action".
And the condition that "the PIP is completely gone".

How about fixing the comment to the following?
Do action after the PIP is completely gone.